.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest, macos-13]
```

Comment:
Pinned? Maybe put a comment in there explaining why so we know in the future whether this is still required or not.